CONTRIBUTING.md

```diff
@@ -6,7 +6,7 @@ information on this and other language SIGs.
 
 See the [public meeting notes](https://docs.google.com/document/d/1CIMGoIOZ-c3-igzbd6_Pnxx1SjAkjwqoYSUWxPY8XIs/edit)
 for a summary description of past meetings. To request edit access, join the
-meeting or get in touch on [Gitter](https://gitter.im/open-telemetry/opentelemetry-python).
+meeting or get in touch on [Slack](https://app.slack.com/client/T08PSQ7BQ/C01NP3BV26R).
```

Comment:
This link doesn't send me anywhere, this one does though: https://cloud-native.slack.com/archives/C01PD4HUVBL

Reply:
i see the other links were updated but this one still looks like the old link?